solution/ui/regulations/eregs-component-lib/src/components/RecentSupplementalContent.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+    <div class="supplemental-content-list">
+        <template
+            v-for="(content, index) in limitedContent"
+        >
+        <div class="category-labels">
+                        <div class="result-label category-label">
+
+            {{content.category.name}}
+                        </div></div>
+            <supplemental-content-object
+                :key="index"
+                :name="content.name"
+                :description="content.description"
+                :date="content.date"
+                :url="content.url"
+            >
+            </supplemental-content-object>
+        </template>
+       
+
+    </div>
+</template>
+
+<script>
+import SupplementalContentObject from "./SupplementalContentObject.vue";
```

Comment:
what was changed here?

Reply:
I'm just going to mark this resolved because I'm not going to push for refactoring these to use the Composition API.  I'd like it to be used going forward, though.